frontend_app_html-document/src/container/HtmlDocument.jsx

```diff
@@ -87,40 +89,59 @@ class HtmlDocument extends React.Component {
 
   // TLM Handlers
   handleContentModified = data => {
-    const { state } = this
+    const { state } = this // TODO review and test localstorage
     if (data.content.content_id !== state.content.content_id) return
 
-    globalThis.tinymce.remove('#wysiwygNewVersion')
-
-    localStorage.removeItem(
-      generateLocalStorageContentId(state.content.workspace_id, state.content.content_id, state.appName, 'rawContent')
-    )
-
     const localStorageComment = localStorage.getItem(
       generateLocalStorageContentId(data.workspace_id, data.content_id, state.appName, 'comment')
     )
 
+    if (state.mode === APP_FEATURE_MODE.EDIT) {
+      if (state.loggedUser.user_id === data.author.user_id) {
+        globalThis.tinymce.remove('#wysiwygNewVersion')
+
+        localStorage.removeItem(
+          generateLocalStorageContentId(state.content.workspace_id, state.content.content_id, state.appName, 'rawContent')
+        )
+
+        this.setState({ mode: APP_FEATURE_MODE.VIEW })
+      } else {
+        this.setState({
+          editionAuthor: data.author.public_name,
+          keepEditingWarning: true
+        })
+      }
+    }
+
     this.setState(prev => ({
       ...prev,
-      mode: APP_FEATURE_MODE.VIEW,
       content: {
         ...prev.content,
         ...data.content
-        // raw_content: data.raw_content // TODO
+        // raw_content: data.raw_content // TODO blocked by back
       },
       newComment: localStorageComment || '',
       rawContentBeforeEdit: prev.content.raw_content
     }))
 
-    this.loadTimeline()
+    this.loadTimeline() // TODO blocked by back
   }
 
   handleContentCreated = data => {
     const { state } = this
-    if (data.content.content_id !== state.content.content_id) return
+    if (data.content.parent_id !== state.content.content_id) return
+
+    if (data.content.content_type === 'comment') { // TODO review and test localstorage (and appContentSaveNewComment at appContentFactory)
+      if (state.timelineWysiwyg) tinymce.get('wysiwygTimelineComment').setContent('')
+
+      localStorage.removeItem(
+        generateLocalStorageContentId(state.content.workspace_id, state.content.content_id, state.config.slug, 'comment')
+      )
+
+      GLOBAL_dispatchEvent({ type: CUSTOM_EVENT.RELOAD_APP_FEATURE_DATA(state.config.slug), data: {} })
```

Comment:
Why do you need this call ?
The tlm should contains all the information required